### ISSUE TITLE
v0.29.0 does not work with Debian 10 because of glibc

### ISSUE THREAD

**Issue by curquiza** [meilisearch/meilisearch#2850]
Use Meilisearch v0.29.0 freshly new released will lead to this following error

```bash
Oct 04 10:15:21 meilisearch-v0-29-0-debian-10-build--04-10-2022-12-07-42 systemd[1]: Started Meilisearch.
Oct 04 10:15:21 meilisearch-v0-29-0-debian-10-build--04-10-2022-12-07-42 meilisearch[530]: /usr/bin/meilisearch: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /usr
Oct 04 10:15:21 meilisearch-v0-29-0-debian-10-build--04-10-2022-12-07-42 systemd[1]: meilisearch.service: Main process exited, code=exited, status=1/FAILURE
Oct 04 10:15:21 meilisearch-v0-29-0-debian-10-build--04-10-2022-12-07-42 systemd[1]: meilisearch.service: Failed with result 'exit-code'.
```

Reported by @alallema, impacts @meilisearch/integration-team

**Comment by curquiza** [comment]
@loiclec @Kerollmops the fix should be done in a branch starting from `release-v0.29.1` and that will be merged into `release-v0.29.1`.
`release-v0.29.1` is already created and starts from `stable`

**Comment by Mcdostone** [comment]
Hi, is there a minimum requirement for the glibc version?

**Comment by Mcdostone** [comment]
Do you think it could be interesting to have have CI tests to detect this issue?

I was planning to update meilisearch from `0.28.1` to `0.29.0` and I had to rollback to `0.28.1` because of my glibc version:
```bash
hostnamectl
  Operating System: Red Hat Enterprise Linux 8.6 (Ootpa)
       CPE OS Name: cpe:/o:redhat:enterprise_linux:8::baseos
            Kernel: Linux 4.18.0-372.26.1.el8_6.x86_64
      Architecture: x86-64


ldd --version
ldd (GNU libc) 2.28
Copyright (C) 2018 Free Software Foundation, Inc.
```

**Comment by curquiza** [comment]
Sorry for the issue on your side

This is something we want to avoid in the future, definitely. We supposed it's due to our CI upgrade from ubuntu-18.04 to ubunto-20.04 because GitHub is deprecated ubuntu-18.04 soon...

**Comment by curquiza** [comment]
📣 The minimal glibc required by Meilisearch 2.27 (on v0.29.1)

We plan to discuss this with the team to provide clear and transparent information about this in the future, and in the official documentation.

**Comment by curquiza** [comment]
Closed by #2862
